src/helm/config/model_metadata.yaml

```diff
@@ -1423,8 +1423,17 @@ models:
     release_date: 2024-01-25
     tags: [TEXT_MODEL_TAG, OPENAI_CHATGPT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG, INSTRUCTION_FOLLOWING_MODEL_TAG]
 
-  # Claimed length is 16,384; we round down to 16,000 for the same reasons as explained
-  # in the openai/gpt-3.5-turbo-0613 comment
+  - name: openai/gpt-3.5-turbo-0125
+    display_name: gpt-3.5-turbo-0125
```

Comment:
Now they can't release two models on the same date!

Reply:
I wish they would use `yyyymmdd` dates to avoid confusion...